tests/Unit/Evolution/Systems/GeneralizedHarmonic/Test_UpwindFlux.cpp

```diff
@@ -0,0 +1,284 @@
+                             shift_ext, inverse_spatial_metric_ext, gamma_1,
+                             gamma_2, minus_unit_normal_one_form_ext);
+
+  // Check that if the same fields are given for the interior and exterior
+  // (except that the normal vector gets multiplied by -1.0) that the
+  // numerical flux reduces to the flux
+  auto psi_normal_dot_numerical_flux = make_with_value<
+      db::item_type<::Tags::NormalDotNumericalFlux<gr::Tags::SpacetimeMetric<
+          spatial_dim, Frame::Inertial, DataVector>>>>(
+      one, std::numeric_limits<double>::signaling_NaN());
+  auto pi_normal_dot_numerical_flux =
+      make_with_value<db::item_type<::Tags::NormalDotNumericalFlux<
+          GeneralizedHarmonic::Tags::Pi<spatial_dim, Frame::Inertial>>>>(
+          one, std::numeric_limits<double>::signaling_NaN());
+  auto phi_normal_dot_numerical_flux =
+      make_with_value<db::item_type<::Tags::NormalDotNumericalFlux<
+          GeneralizedHarmonic::Tags::Phi<spatial_dim, Frame::Inertial>>>>(
+          one, std::numeric_limits<double>::signaling_NaN());
+  ::TestHelpers::NumericalFluxes::apply_numerical_flux(
+      flux_computer, packaged_data_int, packaged_data_int_opposite_normal,
+      make_not_null(&psi_normal_dot_numerical_flux),
+      make_not_null(&pi_normal_dot_numerical_flux),
+      make_not_null(&phi_normal_dot_numerical_flux));
+
+  ::GeneralizedHarmonic::ComputeNormalDotFluxes<spatial_dim>
+      normal_dot_flux_computer{};
+  auto psi_normal_dot_flux = make_with_value<
+      db::item_type<::Tags::NormalDotFlux<gr::Tags::SpacetimeMetric<
+          spatial_dim, Frame::Inertial, DataVector>>>>(
+      one, std::numeric_limits<double>::signaling_NaN());
+  auto pi_normal_dot_flux = make_with_value<db::item_type<::Tags::NormalDotFlux<
+      GeneralizedHarmonic::Tags::Pi<spatial_dim, Frame::Inertial>>>>(
+      one, std::numeric_limits<double>::signaling_NaN());
+  auto phi_normal_dot_flux =
+      make_with_value<db::item_type<::Tags::NormalDotFlux<
+          GeneralizedHarmonic::Tags::Phi<spatial_dim, Frame::Inertial>>>>(
+          one, std::numeric_limits<double>::signaling_NaN());
+  normal_dot_flux_computer.apply(
+      make_not_null(&psi_normal_dot_flux), make_not_null(&pi_normal_dot_flux),
+      make_not_null(&phi_normal_dot_flux), spacetime_metric_int, pi_int,
+      phi_int, gamma_1, gamma_2, lapse_int, shift_int,
+      inverse_spatial_metric_int, unit_normal_one_form_int);
+
+  CHECK_ITERABLE_APPROX(psi_normal_dot_numerical_flux, psi_normal_dot_flux);
+  CHECK_ITERABLE_APPROX(pi_normal_dot_numerical_flux, pi_normal_dot_flux);
+  CHECK_ITERABLE_APPROX(phi_normal_dot_numerical_flux, phi_normal_dot_flux);
+}
+}  // namespace
+
+SPECTRE_TEST_CASE("Unit.Evolution.Systems.GeneralizedHarmonic.UpwindFlux",
+                  "[Unit][Evolution]") {
+  {
+    INFO("test consistency of the upwind flux")
+    test_upwind_flux_consistency_random();
```

Comment:
This function name is misleading since it tests more than consistency. Please remove `consistency` again.